api/inventorypb/agents.proto

```diff
@@ -882,6 +888,12 @@ message AddQANMongoDBProfilerAgentRequest {
   bool tls = 5;
   // Skip TLS certificate and hostname validation.
   bool tls_skip_verify = 6;
+  // Certificate.
+  string tls_certificate_key = 9;
+  // Certificate password.
+  string tls_certificate_key_file_password = 10;
+  // Certificate authority.
+  string tls_ca = 11;
```

Comment:
This is repeated? at the beginning of the block there is a tls_cs field with number 12

Reply:
No, one for AddMongoDBExporterRequest, second one for AddQANMongoDBProfilerAgentRequest and diffrent numbers because of diffrent count of properties.